pkg/kubeadm/tlsutil/tlsutil.go

```diff
@@ -0,0 +1,133 @@
+// Copied from https://github.com/coreos/bootkube/blob/master/pkg/tlsutil/tlsutil.go
+// TODO vendor it properly
+// TODO also see whether some of these should just go into "pkg/util/certificates"
```

Comment:
@aaronlevy @derekparker @gtank @mikedanese ok, so we already have `pkg/util/certificates` and `pkg/crypto`, and now I'm thinking to also vendor this... shall we consolidate all different functions we need for generating certificates and do stuff with them? I'm quite happy with the API this package provides, although it'd be nicer to have handy `.Bytes()` and `.EncodeToPEM()`, or something like that... But I would be also quite happy if we went with something based on `cfssl`, as that provides us gives us config files for all the things, and thereby user can regenerate all the certs out-of-band.